lib/cli/utils.ts

```diff
@@ -4,6 +4,7 @@ import os from 'os';
 import path from 'path';
 
 const SATOSHIS_PER_COIN = 10 ** 8;
+const MAX_DECIMALS = 12;
```

Comment:
I'd prefer what I suggested originally so it is clear what this is for, `MAX_PRICE_DECIMAL_PLACES`